service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/DocumentUploadedEventHandler.java

```diff
@@ -128,7 +119,36 @@ public void sendDocumentsByPost(final FurtherEvidenceUploadedEvent event) {
     }
 
     @EventListener
-    public void sendCourtBundlesToCafcass(final FurtherEvidenceUploadedEvent event) {
+    public void sendCourtBundlesUploadedNotification(final DocumentUploadedEvent event) {
+        final CaseData caseData = event.getCaseData();
```

Comment:
Is there perhaps a way we can move some of this caseDataBefore/After logic to a helper function as it's used in the above function too?